### PR TITLE
Doc: Extending managed cluster model

### PR DESCRIPTION
**Pull request by yue9944882**
> Signed-off-by: yue9944882 <291271447@qq.com>

/cc @qiujian16 @mikeshng

### CODE REVIEW

File: content/en/scenarios/extending-managed-clusters.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+- aggregated resource information (node counts, pod counts)
+- cluster resource watermark/budget
+- any cluster-scoped knowledge of the managed cluster...
+
+
+## Next
+
+After extending your "Managed Cluster" with customized attributes, now we can
+try the advanced cluster selection using the [placement](https://open-cluster-management.io/concepts/placement/)
+policies, which is provided another module of OCM helpful for building your
```
Comment (qiujian16) [inline content/en/scenarios/extending-managed-clusters.md:109]: provided by another module of OCM for building...

Comment (yue9944882) [reply content/en/scenarios/extending-managed-clusters.md:109]: fixed

**Pull request merged**